app/subscriber/src/features/manage-folder/ManageFolder.tsx

```diff
@@ -0,0 +1,178 @@
+import { SearchWithLogout } from 'components/search-with-logout';
+import { DetermineToneIcon } from 'features/home';
+import { trimWords } from 'features/search-page/utils';
+import parse from 'html-react-parser';
+import React from 'react';
+import { DragDropContext, Draggable, Droppable } from 'react-beautiful-dnd';
+import { FaArrowLeft, FaFolderMinus, FaGripLines } from 'react-icons/fa';
+import { useNavigate, useParams } from 'react-router-dom';
+import { useContent } from 'store/hooks';
+import { useFolders } from 'store/hooks/subscriber/useFolders';
+import { Checkbox, Col, IContentModel, IFolderModel, Row } from 'tno-core';
+
+import * as styled from './styled';
+
+/**
+ * Component that renders the manage folder page, allowing user to view/manage content inside a folder
+ * @returns ManageFolder component
+ */
+export const ManageFolder: React.FC = () => {
+  const { id } = useParams();
+  const [, { getFolder, updateFolder }] = useFolders();
+  const navigate = useNavigate();
+  const [, { findContent }] = useContent();
+  const [folder, setFolder] = React.useState<IFolderModel>();
+  const [items, setItems] = React.useState<any>([]);
+  const [selected, setSelected] = React.useState<IContentModel[]>([]);
+
+  /** TODO: Folder content only contains contentId and sortOrder so we have to make an additional call based off of the contentIds to get the headline/summary etc..
+   * assuming we want this to differ eventually.
+   */
+  React.useEffect(() => {
+    getFolder(Number(id)).then((data) => {
+      // have to use temp not folder state as folder state will not update right away
+      const temp = data;
```

Comment:
Instead of this odd variable, just rename your variable in the promise from `data` to `folder`